### PR TITLE
Empty state refactor

### PR DESCRIPTION
**Pull request by RobDolares**
Related Issue: https://github.com/stolostron/backlog/issues/20690
<kbd>Screen Shot 2022-03-21 at 4 43 33 PM</kbd>
<kbd>Screen Shot 2022-03-21 at 4 17 15 PM</kbd>
<kbd>Screen Shot 2022-03-21 at 4 46 53 PM</kbd>
<kbd>Screen Shot 2022-03-21 at 4 02 32 PM</kbd>
<kbd>Screen Shot 2022-03-21 at 4 02 41 PM</kbd>
<kbd>Screen Shot 2022-03-21 at 4 02 48 PM</kbd>
<kbd>Screen Shot 2022-03-21 at 4 03 06 PM</kbd>
<kbd>Screen Shot 2022-03-21 at 4 03 14 PM</kbd>
<kbd>Screen Shot 2022-03-21 at 4 03 22 PM</kbd>
<kbd>Screen Shot 2022-03-21 at 4 03 29 PM</kbd>

### CODE REVIEW

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -1103,12 +1105,12 @@
...
-    "emptystate.credentials.title": "No discovered clusters found",
-    "emptystate.defaultState.msg": "You don't have any discovered clusters. Credentials of <bold>Red Hat OpenShift Cluster Manager</bold> type must be added to configure Discovery.",
-    "emptystate.defaultState.title": "No discovered clusters found",
-    "emptystate.discoveryEnabled.msg": "You don't have any discovered clusters. Cluster discovery was configured. Return to this page later, configure Discovery again, or <a>view documentation <icon /></a>",
-    "emptystate.discoveryEnabled.title": "No discovered clusters found",
+    "emptystate.credentials.msg": "No discovered clusters found. You have {{discoveryConfigTotal}} credentials. Click the <bold>Configure Discovery</bold> button to set up a filter for discovered clusters from your connections.",
+    "emptystate.credentials.title": "You don't have any discovered clusters",
+    "emptystate.defaultState.msg": "Credentials of <italic>Red Hat OpenShift Cluster Manager</italic> type must be added to configure Discovery.  Click the <bold>Add credential</bold> button to add credential.",
```
Comment (KevinFCormier) [inline frontend/public/locales/en/translation.json:1110]: "Click Add credential to add credential" seems a bit redundant

File: frontend/public/locales/en/translation.json [hunk 2]
```diff
@@ -1305,8 +1307,8 @@
     "infraEnv.bulkAction.createInfraEnv": "Create infrastructure environment",
     "infraEnv.createCluster": "Create infrastructure environment",
     "infraEnv.delete.plural": "Delete infrastructure environments",
-    "infraEnv.emptyStateBody": "To create a pool of hosts that is ready for cluster creation, create an infrastructure environment. Once a new infrastructure environment is created, you can discover, provision, and add hosts to it.",
-    "infraEnv.emptyStateHeader": "No infrastructure environments yet",
+    "infraEnv.emptyStateBody": "Click the <bold>Create infrastructure environment</bold> button to create an infrastructure environment. Once a new infrastructure environment is created, you can discover, provision, and add hosts to it.",
```
Comment (KevinFCormier) [inline frontend/public/locales/en/translation.json:1310]: I think "create your resource" would be less redundant here.

**Review by KevinFCormier — COMMENTED** [review]
@RobDolares This looks good. A few strings had the form "Click X to X" and I marked some of them as seeming redundant. Probably the more generic "create your resource" or something more specific about what the result will be would work better in these cases.

Let's ask @swopebe to review the strings. Otherwise, code-wise I think it all looks good.

File: frontend/public/locales/en/translation.json (hunk 1 above)
Comment (RobDolares) [reply frontend/public/locales/en/translation.json:1110]: @KevinFCormier - I completely agree - it felt strange typing it out.  Will definitely heed the advice from @swopebe and doc team to update these.

File: frontend/public/locales/en/translation.json (hunk 2 above)
Comment (RobDolares) [reply frontend/public/locales/en/translation.json:1314]: "Click <bold>Create infrastructure environment</bold> to create your resource.  You can discover, provision, and add hosts to the infrastructure environment after creation."

@swopebe

Comment (swopebe) [reply frontend/public/locales/en/translation.json]: Perfect @RobDolares

**Comment by swopebe** [comment]
you guys are amazing...please let me know how best to keep new team members aware of the work we put in to keep UI content with one voice...I am happy to present again at a meeting. Thanks for keeping us all in line.

**Review by swopebe — DISMISSED** [review]

**Review by zlayne — APPROVED** [review]

**Pull request merged**